index.js

```diff
@@ -60,11 +59,13 @@ require("dotenv").config();
 
 app.use(express.json());
 app.use(cors());
+app.use(jwtMiddleware.unless({ path: ["/"] }));
```

Comment:
We already used `express-unless` on `jwtMiddleware` up on line 24, so I don't think we need to use it again here